.github/workflows/node-tens.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Tests: node.js >= 10'
+
+on: [pull_request, push]
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    uses: ljharb/actions/.github/workflows/node.yml@main
+    with:
+      range: '>= 10'
+      type: minors
+      command: npm run tests-only
+
+  node:
+    name: 'node >= 10'
+    needs: [tests]
+    runs-on: ubuntu-latest
+    steps:
+      - run: 'echo tests completed'
```

Comment:
```suggestion
```

What is this useful for? 🤔

Reply:
It lets you require this one as a required check - otherwise there’s no way to require every matrix job has completed except adding them all one at a time (and there’s a lot)